apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts

```diff
@@ -517,7 +517,7 @@ export class UpdatesViewModel extends Observable {
     let promises = [];
     const files = [];
     // get the max firmware version for each firmware
-    const maxes = mds.reduce((maxes, md) => {
+    const allMaxes = mds.reduce((maxes, md) => {
```

Comment:
```suggestion
    const reducedMaxes = mds.reduce((maxes, md) => {
```
changing name to be consistent with other renames of similar code

Reply:
Make sure to change line 534 if committing this suggestion.